easybuild/easyconfigs/a/AmberTools/AmberTools-21-foss-2021a.eb

```
@@ -11,18 +11,27 @@ description = """AmberTools consists of several independently developed packages
 toolchain = {'name': 'foss', 'version': '2021a'}
 toolchainopts = {'usempi': True}
 
-sources = [SOURCE_TAR_BZ2]
+# download requires registration, see http://ambermd.org/AmberTools17-get.html
+local_download_credentials = '?Name=Easybuild&Institution=Easybuild&City=Internet&State=Other&Country=Belgium'
+source_urls = ['https://ambermd.org/cgi-bin/AmberTools%s-get.pl' % version]
+sources = [{
+    'download_filename': local_download_credentials,
+    'filename': SOURCE_TAR_BZ2,
+}]
 patches = [
     'AmberTools-20_fix_missing_MPI_LIBRARY_error.patch',
     'AmberTools-20_cmake-locate-netcdf.patch',
     'AmberTools-21_CMake-FlexiBLAS.patch',
+    'AmberTools-21_dont_include_config.h_in_top_Makefile.patch',
 ]
 checksums = [
     'f55fa930598d5a8e9749e8a22d1f25cab7fcf911d98570e35365dd7f262aaafd',  # AmberTools-21.tar.bz2
     # AmberTools-20_fix_missing_MPI_LIBRARY_error.patch
     '0b89a0624167bc23876bcdefcb1055f591e38e3bd559a71d5749e342bd311acc',
     '473e07c53b6f641d96d333974a6af2e03413fecef79f879d3fdecf7fecaab4d0',  # AmberTools-20_cmake-locate-netcdf.patch
     '9543812c24c4b7842f64f1f8abaf2c92b5c4c0fadcdbd9811e76b81a778f0d36',  # AmberTools-21_CMake-FlexiBLAS.patch
```

```suggestion
    '9543812c24c4b7842f64f1f8abaf2c92b5c4c0fadcdbd9811e76b81a778f0d36',  # AmberTools-21_CMake-FlexiBLAS.patch
    # AmberTools-20_fix_xblas_missing_make_dependency.patch
    'ff25e91fdc72347a778c3837b581e174d6a8c71efa5b46e11391b18bca84fd65',
```

And the checksum for that patch.

---

All patches added from AT21.12